src/app/view/components/_common/pattern-library/pattern-library.component.html

```diff
@@ -45,7 +45,7 @@ <h3 class="section__title">Data Table</h3>
             </article>
             <h3 class="section__title">Action button</h3>
             <article>
-                <action-button [command]="actionButtonLog"></action-button>
+                 <action-button [command]="actionButtonLog"></action-button>
```

Comment:
Extra space.

Reply:
not sure this extra space is in most of our PR 😄 .